src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AddMissingInterpolationTokenTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Test.Utilities;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.CodeQuality.Analyzers.QualityGuidelines.AddMissingInterpolationTokenAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Microsoft.CodeQuality.Analyzers.UnitTests.QualityGuidelines
+{
+    public class AddMissingInterpolationTokenTests
```

Comment:
Please add the following tests:

- string literal assigned to a variable.
- string that looks like an interpolated string inside a `string.Format` call. Do we want to report? More generally if the string is passed to a `format` parameter do we still want to report?
- cases where you use the `alignment` and `formatString` parts of the tokens (see [doc](https://docs.microsoft.com/dotnet/csharp/language-reference/tokens/interpolated#structure-of-an-interpolated-string))

Reply:
> string that looks like an interpolated string inside a string.Format call. Do we want to report? More generally if the string is passed to a format parameter do we still want to report?

Did you mean something like `string.Format("{0} - {1}", foo, bar)`? If so, that's not something the analyzer should report.

Reply:
More like do we want to report on `string.Format("{x}")` and also `string.Format("{0}")`. I would tend to say no but in this case the code needs to be tweaked to avoid reporting.

Reply:
Currently, the code reports for `string.Format("{x}")` **if** there is a variable called `x` in scope.
But it doesn't report for `string.Format("{0}")`

Reply:
> * cases where you use the `alignment` and `formatString` parts of the tokens (see [doc](https://docs.microsoft.com/dotnet/csharp/language-reference/tokens/interpolated#structure-of-an-interpolated-string))

Could we add tests for these?